azerothcore/azerothcore-wotlk
# feat(DB/Spawns): port from TC spawns and event associations

## Changes Proposed:
-  Ported from TC the related spawns and event associations of NPC Goblin Commoner, and several NPCs involved in the following **events**:
12, 24, 7, 1, 26, 51, 50, 9

About event 9 check if the following NPCs give the related quests

**NPCs questgivers ID**  (check them!):
19148, 18927, 19178, 19177, 19176, 19175, 19169, 19171, 19172, 19173, 20102, 20102

**Quests**
13484, 13484, 13483, 13483, 13483, 13483, 13483, 13484, 13484, 13484, 13484, 13483

related commits
https://github.com/TrinityCore/TrinityCore/commit/1399e705ebbe1429f8549eb41de6e70b823a7b22
https://github.com/TrinityCore/TrinityCore/commit/e6a7818a9a12b085c8ea07cfb8b3c6d1d700a112

## Issues Addressed:
- Closes https://github.com/azerothcore/azerothcore-wotlk/issues/4330
- Closes https://github.com/azerothcore/azerothcore-wotlk/issues/4335

## Tests Performed:
- tested in-game activating and deactivating the event

## How to Test the Changes:
- `.event stop X`
- `.event start X`
- check the NPCs in the related cities (Booty Bay, stormwind etc. )
- check if all the quest givers work properly
- check if the NPCs should stand in all event without any issue
- `.event stop X`

About event 9 check if the following NPCs give the related quests (see the IDs above).
Check also if the NPCs (like goblin commoner) change the equipe when you stop and start a new event.


## Target Branch(es):








 
## How to Test AzerothCore PRs
 
When a PR is ready to be tested, it will be marked as **[WAITING TO BE TESTED]**.

You can help by testing PRs and writing your feedback here in the PR's page on GitHub. Follow the instructions here:

http://www.azerothcore.org/wiki/How-to-test-a-PR
